lib/compression.ts

```diff
@@ -1,11 +1,15 @@
-'use strict';
-const zlib = require('zlib');
-const snappy = require('snappyjs');
+import zlib from 'zlib'
+import snappy from 'snappyjs'
 import { compress as brotliCompress, decompress as brotliDecompress } from 'wasm-brotli'
 
-
+interface PARQUET_COMPRESSION_METHODS {
+  [key:string]: {
+      deflate: Function,
+      inflate: Function
```

Comment:
@wilwade I worked with @waylandli on this and it's not as simple as it might seem, because the union type does not work here, and it doesn't even work to declare the functions as types and use a union of those. So it's going to need more of a refactor than at first blush. I suggest we punt to a later PR.

Reply:
I'm good with that punt. Sounds like the interface there is a bit problematic and might take some code changes (beyond the conversion)